<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1448</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdNumber_M">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>88.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>88</number>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>441</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/overlay/background);</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_S">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>88.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>88</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_dot">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>61</pointsize>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_current">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(172, 175, 178);
selection-background-color: rgb(29, 29, 29);</string>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_current">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(29, 30, 31);</string>
    </property>
    <property name="text">
     <string>    1   2   3   4   5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_T20">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">&lt;u&gt;20&lt;/u&gt;</string>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_T40">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">&lt;u&gt;20&lt;/u&gt;</string>
    </property>
    <property name="text">
     <string>40</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_T60">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">&lt;u&gt;20&lt;/u&gt;</string>
    </property>
    <property name="text">
     <string>60</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_clk">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>203</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="aid.qrc">:/icons/clk</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_bat">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="aid.qrc">:/icons/res/bat.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>282</x>
      <y>168</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="aid.qrc">:/icons/res/test.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_LB2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="aid.qrc">:/icons/LB2</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_LB5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="aid.qrc">:/icons/LB5</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clk">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>440</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/button/res/button_clk.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_record">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>440</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/button/res/button_lock.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_down">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/button/res/button_L.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_up">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>420</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/button/res/button_R.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>130</y>
      <width>634</width>
      <height>488</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="adminTab">
     <attribute name="title">
      <string>Admin</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="adminTabQVBoxLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QGridLayout" name="adminTabQGridLayout">
          <item row="2" column="2">
           <widget class="QComboBox" name="comboBox_waveType">
            <item>
             <property name="text">
              <string>Alpha</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Betta</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Gamma</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1" alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="chargeAdminButton">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
background-color: #ffd300;
border-radius: 15px;
}

QPushButton:pressed {
	background-color: green;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="spinBox_batteryLevel">
            <property name="minimumSize">
             <size>
              <width>82</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>82</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="value">
             <double>50.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="applyToSkinAdminLabel">
            <property name="minimumSize">
             <size>
              <width>518</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Apply to Skin</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="applyToSkinAdminLabel_4">
            <property name="minimumSize">
             <size>
              <width>518</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Frequence select</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="applyToSkinAdminLabel_2">
            <property name="minimumSize">
             <size>
              <width>518</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Wave</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="batteryLevelAdminLabel">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Battery Level</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pushButton_errorTrigger">
            <property name="text">
             <string>Error trigger</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QComboBox" name="comboBox_freSel">
            <item>
             <property name="text">
              <string>0.5Hz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>77Hz</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100Hz</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="applyToSkinAdminLabel_3">
            <property name="minimumSize">
             <size>
              <width>518</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Device error</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="checkBox_applySkin">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox::indicator {
width: 50px;
height: 30px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
	border-image: url(:/button/powerOFF);
}
QCheckBox::indicator:checked {
	border-image: url(:/button/powerON);
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="3">
           <widget class="QTableView" name="tableView"/>
          </item>
          <item row="6" column="2">
           <widget class="QPushButton" name="pushButton_clearHistory">
            <property name="text">
             <string>Clear History</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox_power">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {
width: 50px;
height: 30px;
color: rgb(0, 0, 0);
}
QCheckBox::indicator:unchecked {
	border-image: url(:/button/powerOFF);
}
QCheckBox::indicator:checked {
	border-image: url(:/button/powerON);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_mask">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>441</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/overlay/background);</string>
    </property>
    <property name="interactive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_waveType">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>375</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="aid.qrc">:/icons/res/alpha.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_Freq">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>380</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(29, 30, 31);</string>
    </property>
    <property name="text">
     <string>0.5Hz</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>label_Freq</zorder>
   <zorder>label_dot</zorder>
   <zorder>lcdNumber_S</zorder>
   <zorder>lcdNumber_M</zorder>
   <zorder>progressBar_current</zorder>
   <zorder>label_current</zorder>
   <zorder>label_T20</zorder>
   <zorder>label_T40</zorder>
   <zorder>label_T60</zorder>
   <zorder>label_clk</zorder>
   <zorder>label_bat</zorder>
   <zorder>label_LB2</zorder>
   <zorder>label_LB5</zorder>
   <zorder>tabWidget</zorder>
   <zorder>label_waveType</zorder>
   <zorder>graphicsView_mask</zorder>
   <zorder>label_error</zorder>
   <zorder>checkBox_power</zorder>
   <zorder>pushButton_clk</zorder>
   <zorder>pushButton_down</zorder>
   <zorder>pushButton_up</zorder>
   <zorder>pushButton_record</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1448</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="aid.qrc"/>
 </resources>
 <connections/>
</ui>
